dev/integration_tests/ios_host_app/Host.xcodeproj/project.pbxproj

```diff
@@ -447,6 +447,7 @@
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
```

Comment:
Host apps need to add this to work on arm64 macOS.  That's not new to this PR, but this PR actually makes it fail on a non-Apple Silicon device, like our test agents.